libraries/engage/favorites/components/Item/Item.jsx

```diff
@@ -311,17 +311,17 @@ const FavoriteItem = ({
   return (
     <ProductListEntryProvider productId={product.id}>
       <SurroundPortals portalName={FAVORITES_LIST_ITEM} portalProps={product}>
-        <div className={styles.root}>
+        <div className={classNames(styles.root, 'engage__favorites__item')}>
           <Link
             className={styles.imageContainer}
             component="div"
             href={productLink}
             aria-hidden
           >
-            <ProductImage src={product.featuredImageBaseUrl} resolutions={gridResolutions} />
+            <ProductImage className={classNames('engage__favorites__item__image')} src={product.featuredImageBaseUrl} resolutions={gridResolutions} />
```

Comment:
could you please add an additional class to the `Link` component (e.g. "engage__favorites__item__image-container")? maybe someone needs to apply custom styling to all contents of the left row.

Reply:
done